edge-hub/core/test/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter.Test/DeviceProxyTest.cs

```diff
@@ -49,16 +51,17 @@ public async Task DesiredUpdateCallForwarded()
                 .Setup(h => h.SendDesiredPropertiesUpdate(It.Is<IMessage>(m => m == desired), It.Is<IIdentity>(i => i == identity)))
                 .Returns(Task.CompletedTask);
 
-            var sut = new DeviceProxy(identity, twinHandler, m2mHandler, c2dHandler, directMethodHandler);
+            var sut = new DeviceProxy(identity, connectionHanlder, twinHandler, m2mHandler, c2dHandler, directMethodHandler);
 
             await sut.OnDesiredPropertyUpdates(desired);
 
-            Mock.Get(directMethodHandler).VerifyAll();
+            Mock.Get(twinHandler).VerifyAll();
```

Comment:
Why is this changed to twinHandler?

Reply:
it was a copy-paste error from earlier and it did not call the "verifyall" on the set up mock object. Now it checks the right mock. Earlier the test passed only because nothing was setup on 'directMethodHandler', so verifyAll meant 'verify nothing' - which always passed.